invenio_oauthclient/templates/invenio_oauthclient/signup.html

```diff
@@ -22,31 +22,31 @@
 {% from "invenio_oauthclient/_macros.html" import render_field, form_errors %}
 
 {% block page_body %}
-<div class="row">
+  <div class="row">
     {%- block signup_panel %}
     <div class="col-md-6 well well-lg col-md-offset-3">
-    {%- block signup_header %}
-    {%- block signup_app_icon %}
-    {% if app_icon -%}
-    <span class="pull-right"><i class="{{app_icon}} fa-5x"></i></span>
-    {%- endif %}
-    {%- endblock %}
-    <h3> {{ _('Sign-up with %(title)s!', title=app_title) }}</h3>
-    <p>{{ _('Fill in your details to complete your registration. You only have to do this once.') }}</p>
-    <hr />
-    {% endblock %}
-    {%- block signup_form %}
-        <form name="edit" method="post" role="form">
-          {%- for field in form %}
-            {{ render_field(field) }}
-          {%- endfor %}
-          <div class="form-actions">
-            <a href="/" class="btn btn-default"><i class="fa fa-times"></i> Cancel</a>
-            <button type="submit" name="continue" class="btn btn-warning"/><i class="fa fa-edit"></i> {{ _("Sign-up") }}</button>
-          </div>
-        </form>
+      {%- block signup_header %}
+      {%- block signup_app_icon %}
+      {% if app_icon -%}
+        <span class="pull-right"><i class="{{app_icon}} fa-5x"></i></span>
+        {%- endif %}
         {%- endblock %}
+        <h3> {{ _('Sign-up with %(title)s!', title=app_title) }}</h3>
+        <p>{{ _('Fill in your details to complete your registration. You only have to do this once.') }}</p>
+        <hr />
+      {% endblock %}
+      {%- block signup_form %}
+      <form name="edit" method="post" role="form">
+        {%- for field in form %}
+        {{ render_field(field) }}
+        {%- endfor %}
+        <!-- <div class="form-actions"> -->
```

Comment:
sorry, I forgot to remove.
I removed because I'm using the signup form that already have a "register" button.